webapp/src/components/ParcelDetailPage/ParcelDetailPage.container.js

```diff
@@ -13,14 +14,16 @@ import ParcelDetailPage from './ParcelDetailPage'
 const mapState = (state, ownProps) => {
   const parcel = ownProps.asset
   const getParcelMortgage = getParcelMortgageFactory(parcel.x, parcel.y)
+  const bids = getWalletBidsByAsset(state, ownProps.asset, ASSET_TYPES.parcel)
+  const archivedBids = getArchivedBids(state)
```

Comment:
**To discuss**

Maybe this: 

```
  const bids = getWalletBidsByAsset(state, ownProps.asset, ASSET_TYPES.parcel)
  const archivedBids = getArchivedBids(state)
  bids.filter(bid => !archivedBids[bid.id])
```

can be a selector in archiveBid?

getWalletArchivedBidsByAsset